samples/bpf/xdp_ssl_sni_drop_all.c

```diff
@@ -0,0 +1,28 @@
+#define KBUILD_MODNAME "foo"
```

Comment:
what is this for? BPF code isn't a module right?

Reply:
Back then we though it would be better to just put it there because of this: https://github.com/torvalds/linux/commit/96a8eb1eeed2c3485cdba198fab3a2faaec386d3. However, seeing that some samples don't have this, I think we should research a bit more and see it this is indeed necessary.